backend/plugins/posclient_api/src/modules/posclient/graphql/resolvers/customResolvers/cover.ts

```
@@ -0,0 +1,16 @@
+import { IContext } from '@/posclient/@types/types';
+import { ICover } from '~/modules/posclient/@types/cover';
+
+export default {
+  async user(order: ICover, {}, { models }: IContext) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)